staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -304,41 +303,14 @@ var (
 			},
 		},
 	)
-	SuggestedFlowSchemaSystemLeaderElection = newFlowSchema(
-		"system-leader-election", "leader-election", 100,
```

Comment:
would cluster admins have references to the old name under normal circumstances? (e.g. to reference it from another object or to modify it in some way?)